compiler/damlc/daml-ide-core/dlint.yaml

```diff
@@ -117,12 +117,12 @@
     - hint: {lhs: "[] == x", rhs: null x}
     - hint: {lhs: length x /= 0, rhs: not (null x), name: Use null}
     - hint: {lhs: 0 /= length x, rhs: not (null x), name: Use null}
-    - hint: {lhs: "\\x -> [x]", rhs: "(:[])", name: "Use :"}
+    - hint: {lhs: "\\x -> [x]", rhs: "(::[])", name: "Use :"}
     - warn: {lhs: map (uncurry f) (zip x y), rhs: zipWith f x y}
     - hint: {lhs: map f (zip x y), rhs: zipWith (curry f) x y, side: isVar f}
     - warn: {lhs: not (elem x y), rhs: notElem x y}
     - hint: {lhs: foldr f z (map g x), rhs: foldr (f . g) z x, name: Fuse foldr/map}
-    - warn: {lhs: "x ++ concatMap (' ':) y", rhs: "unwords (x:y)"}
+    - warn: {lhs: "x ++ concatMap (' '::) y", rhs: "unwords (x::y)"}
```

Comment:
That hint is wrong. `unwords` operates on `Text` which is not a list in `DAML`.

Reply:
Removed the rule